hicexplorer/HiCMatrix.py

```diff
@@ -278,7 +282,7 @@ def getDekkerBins(self, fileName):
         self.header = []
         try:
             for line in gzip.open(fileName, 'r').readlines():
-                if line.startswith("#"):
+                if line.startswith(b"#"):
```

Comment:
I am looking forward to find another solution, I do not like it too. The issue is that in Python 2 everything is a string (type <str>) but in Python3 sometimes it is a str (class <str>), a bytes object or a numpy.bytes_ object. If it is a str you can handle it like in Python 2 but for a byte and a np.bytes_ you need this leading `b` that the parameter of e.g. `startswith` is a bytes object too.